.github/workflows/tests.yml

```diff
@@ -29,8 +29,8 @@ jobs:
         python-version: '3.9'
     - name: Install deps
       uses: snok/install-poetry@v1
-      env:
-        POETRY_VIRTUALENVS_CREATE: false
+      with:
+        virtualenvs-create: false
```

Comment:
I copied this value from somewhere else but didn't fully understand. do you know why this has to be 'false'?

Reply:
No I don't know. I'm also not sure if calling 'poetry install' here will cause any downsides? It fixed the workflow issues.